src/components/AppBar/NavBar.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useMemo } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import Logo from 'assets/images/logo';
+import Hidden from '@material-ui/core/Hidden';
+import * as Style from './NavBar.style';
+import { DonateButtonWithoutFade, DonateButtonWithFade } from './DonateButton';
+import MobileMenu from './MobileMenu';
+import { StyledMobileMenu } from './AppBar.style';
+import MenuIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+
+const isHomeOrLocationPage = (pathname: string) =>
+  pathname === '/' ||
+  pathname.includes('/us') ||
+  ['/alert_signup', '/compare'].includes(pathname);
+
+const NavBar: React.FC = () => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const closeMenu = () => setMenuOpen(false);
+
+  // We only fade the donate button on the home page on mobile, where the donate
+  // button doesn't appear until the banner is scrolled away.
+  const { pathname } = useLocation();
+  const MobileDonateButton = useMemo(() => {
+    return isHomeOrLocationPage(pathname)
```

Comment:
this `isHomeOrLocationPage` mixes up the original logic a little bit. `DonateButtonWithFade` only applies to homepage paths (not location page paths). i think we can just keep the original logic? ::

```
  // Fade functionality for donate button only applies to homepage paths:
  const pathsToHomepage = ['/', '/alert_signup', '/compare'];
  const buttonWithFade =
    includes(pathsToHomepage, location.pathname) &&
    !location.pathname.includes('us');
```